src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/11 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarnings.sql

```diff
@@ -22,7 +22,7 @@ SELECT distinct pepm.CommitmentId,
     ae.PriceEpisodeFundLineType,
```

Comment:
This query is very inefficient - the query plan shows table scans, filtering, hash matches and sorts that could be rationalised by rewriting the query.